### ISSUE TITLE
[Bug] Gruut tests are failing after new release 

### ISSUE THREAD

**Issue by erogol** [coqui-ai/TTS#1407]
@synesthesiam  Gruut tests started to fail after your new release. Would you mind checking it?

If you run ```make test_text```, you get 

```
nosetests tests.text_tests -x --with-cov -cov  --cover-erase --cover-package TTS tests.text_tests --nologcapture --with-id
........................E
======================================================================
ERROR: test_phonemize (tests.text_tests.test_phonemizer.TestGruutPhonemizer)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/data/TTS/tests/text_tests/test_phonemizer.py", line 121, in test_phonemize
    phonemes = self.phonemizer.phonemize(text, separator="|")
  File "/data/TTS/TTS/tts/utils/text/phonemizers/base.py", line 133, in phonemize
    p = self._phonemize(t, separator)
  File "/data/TTS/TTS/tts/utils/text/phonemizers/gruut_wrapper.py", line 110, in _phonemize
    return self.phonemize_gruut(text, separator, tie=False)
  File "/data/TTS/TTS/tts/utils/text/phonemizers/gruut_wrapper.py", line 77, in phonemize_gruut
    for sentence in gruut.sentences(text, lang=self.language, espeak=self.use_espeak_phonemes):
  File "/root/miniconda3/lib/python3.9/site-packages/gruut-2.0.4-py3.9.egg/gruut/__init__.py", line 79, in sentences
    graph, root = text_processor(text, lang=lang, ssml=ssml, **process_args)
  File "/root/miniconda3/lib/python3.9/site-packages/gruut-2.0.4-py3.9.egg/gruut/text_processor.py", line 432, in __call__
    return self.process(*args, **kwargs)
  File "/root/miniconda3/lib/python3.9/site-packages/gruut-2.0.4-py3.9.egg/gruut/text_processor.py", line 891, in process
    if pipeline_transform(self._transform_date, graph, root):
  File "/root/miniconda3/lib/python3.9/site-packages/gruut-2.0.4-py3.9.egg/gruut/utils.py", line 305, in pipeline_transform
    if transform_func(graph, leaf_node):
  File "/root/miniconda3/lib/python3.9/site-packages/gruut-2.0.4-py3.9.egg/gruut/text_processor.py", line 1764, in _transform_date
    date = dateparser.parse(word.text, **dateparser_kwargs)
  File "/root/miniconda3/lib/python3.9/site-packages/dateparser-1.0.0-py3.9.egg/dateparser/conf.py", line 89, in wrapper
    return f(*args, **kwargs)
  File "/root/miniconda3/lib/python3.9/site-packages/dateparser-1.0.0-py3.9.egg/dateparser/__init__.py", line 54, in parse
    data = parser.get_date_data(date_string, date_formats)
  File "/root/miniconda3/lib/python3.9/site-packages/dateparser-1.0.0-py3.9.egg/dateparser/date.py", line 421, in get_date_data
    parsed_date = _DateLocaleParser.parse(
  File "/root/miniconda3/lib/python3.9/site-packages/dateparser-1.0.0-py3.9.egg/dateparser/date.py", line 178, in parse
    return instance._parse()
  File "/root/miniconda3/lib/python3.9/site-packages/dateparser-1.0.0-py3.9.egg/dateparser/date.py", line 182, in _parse
    date_data = self._parsers[parser_name]()
  File "/root/miniconda3/lib/python3.9/site-packages/dateparser-1.0.0-py3.9.egg/dateparser/date.py", line 196, in _try_freshness_parser
    return freshness_date_parser.get_date_data(self._get_translated_date(), self._settings)
  File "/root/miniconda3/lib/python3.9/site-packages/dateparser-1.0.0-py3.9.egg/dateparser/date.py", line 234, in _get_translated_date
    self._translated_date = self.locale.translate(
  File "/root/miniconda3/lib/python3.9/site-packages/dateparser-1.0.0-py3.9.egg/dateparser/languages/locale.py", line 131, in translate
    relative_translations = self._get_relative_translations(settings=settings)
  File "/root/miniconda3/lib/python3.9/site-packages/dateparser-1.0.0-py3.9.egg/dateparser/languages/locale.py", line 158, in _get_relative_translations
    self._generate_relative_translations(normalize=True))
  File "/root/miniconda3/lib/python3.9/site-packages/dateparser-1.0.0-py3.9.egg/dateparser/languages/locale.py", line 172, in _generate_relative_translations
    pattern = DIGIT_GROUP_PATTERN.sub(r'?P<n>\d+', pattern)
  File "/root/miniconda3/lib/python3.9/site-packages/regex-2022.3.15-py3.9-linux-x86_64.egg/regex/regex.py", line 700, in _compile_replacement_helper
    is_group, items = _compile_replacement(source, pattern, is_unicode)
  File "/root/miniconda3/lib/python3.9/site-packages/regex-2022.3.15-py3.9-linux-x86_64.egg/regex/_regex_core.py", line 1736, in _compile_replacement
    raise error("bad escape \\%s" % ch, source.string, source.pos)
regex._regex_core.error: bad escape \d at position 7
```

**Comment by fijipants** [comment]
Made a pull request [in gruut](https://github.com/rhasspy/gruut/pull/30) to update the broken dependency.

**Comment by erogol** [comment]
Fixed by #1419
